cli/tests/pcluster/validators/test_tags_validators.py

```diff
@@ -100,6 +100,49 @@
             " ['key1'] and will be overridden by the value set in `SlurmQueue/ComputeResources/Tags` for "
             "ComputeResource 'dummy_compute_resource' in queue 'dummy_queue'.",
         ),
+        (
+            [Tag(f"key{i}", f"value{i}") for i in range(0, 41)],
+            None,
+            None,
+            "The number of tags (41) associated with ComputeResource 'dummy_compute_resource' in queue 'dummy_queue' "
+            "has exceeded the limit of 40.",
+        ),
+        (
+            [Tag(f"key{i}", f"value{i}") for i in range(0, 41)],
+            None,
+            None,
+            "The number of tags (41) associated with ComputeResource 'dummy_compute_resource' in queue 'dummy_queue' "
+            "has exceeded the limit of 40.",
+        ),
```

Comment:
What's the difference between the 2 test cases above?

Reply:
They are the same, removed one